docs/guide.adoc

```diff
@@ -270,12 +270,46 @@ players -> for_each' update_player
 
 === map
 
-// Gustav
+`map` applies a function on every element in a list and replaces each element
+with the return value.
+
+[source, sylt]
+----
+l = [1, 2, 3] -> map' fn a: int -> int do a * 2 end
+
+l <=> [2, 4, 6]
+----
+
+[source, sylt]
+----
+points := [1, 2, 3]
+
+sum := 0
+points -> for_each' fn p: int do sum += p end
+
+points_str := points -> map' fn p: int -> str do
+    as_str(p) + "/" as_str(sum)
+end
+
+points_str <=> ["1/6", "2/6", "3/6"]
+----
 
 === filter
 
+`filter` applies a function on every element in a list and keeps it iff the
+function returns true.
```

Comment:
I do like my iffs as much as the next guy - but this isn't a mathematical proof - so what use do we get from it other than raising the bar of understanding?